site/components/pages/Resources/Container/styles.ts

```diff
@@ -0,0 +1,61 @@
+import { css } from "@emotion/css";
+import breakpoints from "@klimadao/lib/theme/breakpoints";
+
+export const layout = css`
+  flex-direction: row;
+  justify-content: center;
+  padding: 0rem 3.4rem;
+  width: 100%;
+  ${breakpoints.medium} {
+    display: flex;
+  }
+`;
+
+export const spacing = css`
+  width: 100%;
+  max-width: var(--site-max-width);
+`;
+
+export const resourcesNavigation = css`
+  position: absolute;
+  margin-top: 80px;
+`;
+
+export const list = css`
+  list-style: none;
+  background-color: #ffffff;
+  border-radius: 8px;
+  padding: 5px;
+  width: 160px;
+  height: 140px;
+  box-shadow: 0px 4px 28px rgba(0, 0, 0, 0.06);
+`;
+
+export const listItem = css`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 44px;
+  padding: 12px;
```

Comment:
we need to use `rem` everywhere so that the UI scales evenly as the user's font-size does.

I did a reset on the root `html` so that 1rem = 10px